flight_stack:
  flight_control:
    ros__parameters:
      agent_node_name: /flight_stack/microRTPS_agent
      data_to_px4: [true, true, false, false, false, false]
      fmu_command_attempts: 3
      fmu_command_timeout: 5000
      landing_step: 0.2
      landing_timeout: 20000
      link_namespace: drone/
      low_battery_voltage: 21.0
      monitor_battery: true
      odometry_topic_name: /drone/odometry
      roll_pitch_stabilization_confidence: 0.087266
      setpoints_period: 50
      takeoff_position_confidence: 0.1
      takeoff_timeout: 40000
      tf2_timeout: 0.1
      travel_sleep_time: 200
      turn_sleep_time: 100
      turn_step: 0.174532925 # 10°
      update_setpoint: false
      use_sim_time: false
      v_horz_stabilization_max: 0.1
      v_vert_stabilization_max: 0.1
      yaw_stabilization_confidence: 0.087266 # 5°
